src/lib/Format.js

```diff
@@ -37,3 +37,36 @@ exports.niceNumDecimals = function niceNumDecimals(input) {
   }
   return 7;
 };
+
```

Comment:
Relative times are good for chats and social media where relative time is important but exact time is irrelevant. For financial applications, relative time is unimportant but exact time is important.

Lets keep things simple and just show the exact time relative to the browser's timezone